src/parser/mod.rs

```diff
@@ -12289,6 +12352,137 @@ impl<'a> Parser<'a> {
         }
         false
     }
+
+    /// Look for an expected keyword, without consuming it
+    fn peek_keyword(&self, expected: Keyword) -> bool {
+        match self.peek_token().token {
+            Token::Word(w) => expected == w.keyword,
+            _ => false,
+        }
+    }
+
+    /// Look for one of expected keyword, without consuming it
+    fn peek_keywords(&self, expected: &[Keyword]) -> bool {
```

Comment:
```suggestion
    fn peek_one_of_keywords(&self, expected: &[Keyword]) -> bool {
```
since parse_keywords might instead suggest that we expect to parse the specified keywords in sequence